Update Book Dash chapter to clarify who our target audience is

malvikasharan opened the pull request:
<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

<!-- Describe the problem you're trying to fix in this pull request. Please reference any related issue and use fixes/close to automatically close them, if pertinent. For example: "Fixes #58", or "Addresses (but does not close) #238". -->

* Clarifying the eligibility to apply to The BOok Dash

### List of changes proposed in this PR (pull-request)

- Update landing page to add the eligibility early on
- Update application and review page to add template and update info to include different sections currently added to the form
- Replace 'development' - 'contribution' session as suggested by @EstherPlomp 

<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here --> 
### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

- [ ] Everything looks ok?

### Acknowledging contributors

<!-- Please select the correct box -->

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-selection.md:
```diff
@@ -13,7 +13,7 @@ These tasks usually include the following roles and responsibilities:
 - **Selection meeting**: participate in the selection meeting to help ensure that the selection is equitable (time commitment: 1.5-2 hours)
 - **Organisation meeting**: attend 1 call to finalise the logistics for the organisation (program agenda and participant support will be carried out by The Turing Way core members) (time commitment: 1 hour)
-- **Session host and mentorship**: lead one of the 2 onboarding calls or one GitHub introduction session before and 1-2 of the development sessions or social sessions during the Book Dash based on their schedule (three 1.5 to 2.5 hour-long sessions)
+- **Session host and mentorship**: lead one of the 2 onboarding calls or one GitHub introduction session before and 1-2 of the collaboration sessions or social sessions during the Book Dash based on their schedule (three 1.5 to 2.5 hour-long sessions)
```
```suggestion
- **Session host and mentorship**: lead one of the 2 onboarding calls or one GitHub introduction session before and 1-2 of the contribution sessions or social sessions during the Book Dash based on their schedule (three 1.5 to 2.5 hour-long sessions)
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -2,7 +2,7 @@
 
 This subchapter provides information on the events that are organised for the book dash attendees.
-These events include a pre-event webinar for information, onboarding calls, GitHub introduction, development (coworking) sessions during the Book Dash and Community Share-Outs, which are explained below.
+These events include a pre-event webinar for information, onboarding calls, GitHub introduction, collaboration (coworking) sessions during the Book Dash and Community Share-Outs, which are explained below.
```
```suggestion
These events include a pre-event webinar for information, onboarding calls, GitHub introduction, contribution sessions during the Book Dash and Community Share-Outs, which are explained below.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
 ## During the Book Dash
 
-### Development Sessions
+### Collaboration Sessions
```
```suggestion
### Contribution Sessions
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
-On each day of Book Dash, we will hold three 2.5 hour-long development sessions, when hosting virtually each followed by a break or social event.
-When hosting an in-person Book Dash, four 75 minutes long development sessions will be facilitated, each separated by 15 minutes for report out followed by a break.
-Each development session will follow the Pomodoro technique that we also use for the {ref}`coworking calls<ch-coworking-motivation>` using the [browser-based shared (cuckoo) clock](https://cuckoo.team/tw-bookdash) to coordinate the time.
+On each day of Book Dash, we will hold three 2.5 hour-long collaboration sessions when hosting virtually each followed by a break or social event.
+When hosting an in-person Book Dash, four 75 minutes long collaboration sessions will be facilitated, each separated by 15 minutes for report out followed by a break.
+Each collaboration session will follow the Pomodoro technique for coworking that we also use for the {ref}`coworking calls<ch-coworking-motivation>` using the [browser-based shared (cuckoo) clock](https://cuckoo.team/tw-bookdash) to coordinate the time.
```
```suggestion
On each day of Book Dash, we will hold three 2.5 hour-long contribution sessions when hosting virtually each followed by a break or social event.
Each contribution session will follow the Pomodoro technique for coworking that we also use for the {ref}`coworking calls<ch-coworking-motivation>` using the [browser-based shared (cuckoo) clock](https://cuckoo.team/tw-bookdash) to coordinate the time.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
 Participants are also encouraged to come up with the main sections of their contributions and collect their thoughts in bullet points on a GitHub issue.
 
-The rest of the development sessions throughout the Book Dash will be facilitated in breakouts allowing members with similar ideas to collaborate.
+The rest of the collaboration sessions throughout the Book Dash will be facilitated in breakouts allowing members with similar ideas to collaborate.
```
```suggestion
The rest of the contribution sessions throughout the Book Dash will be facilitated in breakouts allowing members with similar ideas to collaborate.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
 This is also a chance for everyone to meet the participants and helpers informally, learn about each other and connect with specific members who plan to work on the related topic during the book dash.
 Participants will be supported to collaborate in small groups, organising their thoughts and ideas for contributions during or after the Book Dash.
-Each development session will end with participants sharing their work with others and identifying new collaborators based on the similarity of their interests.
+Each collaboration session will end with participants sharing their work with others and identifying new collaborators based on the similarity of their interests.
```
```suggestion
Each contribution session will end with participants sharing their work with others and identifying new collaborators based on the similarity of their interests.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
 ### Social Sessions
 
-There will also be themed social discussion or networking session where participants can interact with others, continue their discussions outside the development session or take a break as it's most sustainable and helpful for their participation.
+There will also be themed social discussion or networking session where participants can interact with others, continue their discussions outside the collaboration session or take a break as it's most sustainable and helpful for their participation.
```
```suggestion
There will also be themed social discussion or networking session where participants can interact with others, continue their discussions outside the contribution session or take a break as it's most sustainable and helpful for their participation.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-events.md:
```diff
@@ -46,35 +46,35 @@ In the previous events, we have used the following materials:
 
 At the end of each day, participants can gather all their contributions on _The Turing Way_ GitHub repository (issues, pull requests and notes) and add that in the shared notes.
-Specifically, during the last development session, they will be asked to add their details to the [contributors](https://github.com/alan-turing-institute/the-turing-way/tree/book-dash-chapter/contributors.md) file and provide anonymous feedback on a {ref}`Pluses and Delta<ch-template-bookdash-feedback>` notes.
+Specifically, during the last collaboration session, they will be asked to add their details to the [contributors](https://github.com/alan-turing-institute/the-turing-way/tree/book-dash-chapter/contributors.md) file and provide anonymous feedback on a {ref}`Pluses and Delta<ch-template-bookdash-feedback>` notes.
```
```suggestion
Specifically, during the last contribution session, they will be asked to add their details to the [contributors](https://github.com/alan-turing-institute/the-turing-way/tree/book-dash-chapter/contributors.md) file and provide anonymous feedback on a {ref}`Pluses and Delta<ch-template-bookdash-feedback>` notes.
```

malvikasharan commented on book/website/community-handbook/bookdash/bookdash-application.md:
```diff
@@ -7,41 +7,46 @@ In this subchapter, we have provided details on the book dash's application and
+This allows the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.
 
-To fairly make this selection, a call for applications is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
+To fairly make this selection, a call for applications is opened at least 2 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```
```suggestion
To fairly make this selection, a call for applications is opened at least two months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```

malvikasharan reviewed (commented):
Updated 'Collaboration' -> 'Contribution' session

aleesteele commented on book/website/community-handbook/bookdash.md:
```diff
@@ -7,7 +7,8 @@ They also contribute to enhancing the project by improving the ways we work in t
 However, to ensure that international participants have an equal chance to join and address the challenges of hosting in-person events during the COVID-19 pandemic, we started hosting Book Dashes virtually. 
 These virtual Book Dashes are five days long, designed for flexible participation by members in different time zones. 
-Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short co-working sessions each day based on their availability.
+Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called contribution sessions each day based on their availability.
```
```suggestion
Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called "contribution sessions" each day based on their availability.
```
Just adding this suggestion, if this is what they are officially called :)

malvikasharan replied:
Good catch Anne.

malvikasharan commented on book/website/community-handbook/bookdash.md:
```diff
@@ -7,7 +7,8 @@ They also contribute to enhancing the project by improving the ways we work in t
 These virtual Book Dashes are five days long, designed for flexible participation by members in different time zones. 
-Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short co-working sessions each day based on their availability.
+Meaning, rather than committing their entire working day, participants can choose one or multiple 2.5 hours short collaborative co-working calls, called "contribution sessions" each day based on their availability.
+In the future, we would like to co-design hybrid events with "virtual-first" approach for the Book Dash and additional opportunity for participants to organise small local meet-ups for collaboration and social events.
```
```suggestion
In the future, we will co-design hybrid events with "online-first" approach for the Book Dash attendees who will have the opportunity to organise small local meet-ups for collaboration and social events.
```

RaoOfPhysics commented:
OK, looks like https://github.com/alan-turing-institute/the-turing-way/pull/2303#issuecomment-1074009197 points to the main page!! Shall we merge, @malvikasharan? :D

malvikasharan commented:
All check passed!! Please merge it and I will explain this in the issue.

malvikasharan commented:
In this PR we started to have deployment errors due to jinja's depricated package that I see Chris has recently addressed in Jupyter Book: https://github.com/executablebooks/jupyter-book/issues/1676.

I pulled the recent changes and added Jinja package version in the requirement file.

There were few more label errors that I have fixed in this PR to allow deployment to pass erorr-free. Everything seems t work fine now.

da5nsy commented:
My level of git understanding is minimal - but this merge seems to have added 100mb (750mb -> 850mb) to the repo. Was this intentional?
